code/modules/food_and_drink/plants.dm

```diff
@@ -509,8 +509,8 @@
 	crop_prefix = "bowling "
 	desc = "Just keep rollin' rollin'."
 	icon = 'icons/obj/items.dmi'
```

Comment:
It seems that this should be `icons/obj/foodNdrink/food_produce.dmi` now.

Reply:
Fixed!